### PR TITLE
Fixed branch name in PR created succesfully message

### PR DESCRIPTION
**Pull request by jswietek**
Changed target name of successful PR creation message from <owner>/<target_branch_name> (e.g. jswietek/master) to <owner>/<repo_name> (e.g. jswietek/VisualStudio)

### CODE REVIEW

File: src/GitHub.App/ViewModels/PullRequestCreationViewModel.cs [hunk 1]
```diff
@@ -111,9 +111,9 @@ public class PullRequestCreationViewModel : BaseViewModel, IPullRequestCreationV
 
             .OnExecuteCompleted(pr =>
             {
-                notifications.ShowMessage(String.Format(CultureInfo.CurrentCulture, Resources.PRCreatedUpstream, TargetBranch.Id,
-                    TargetBranch.Repository.CloneUrl.ToRepositoryUrl().Append("pull/" + pr.Number)));
-            });
+				notifications.ShowMessage(String.Format(CultureInfo.CurrentCulture, Resources.PRCreatedUpstream, TargetBranch.Repository.Owner + "/" + TargetBranch.Repository.Name,
+					TargetBranch.Repository.CloneUrl.ToRepositoryUrl().Append("pull/" + pr.Number)));
+			});
```
Comment (shana) [inline src/GitHub.App/ViewModels/PullRequestCreationViewModel.cs]: Could you fix the indentation here please?

Comment (tocsoft) [inline src/GitHub.App/ViewModels/PullRequestCreationViewModel.cs]: looks like its a tabs vs spaces issue.

File: src/GitHub.App/ViewModels/PullRequestCreationViewModel.cs [hunk 2]
```diff
@@ -108,10 +108,9 @@ public class PullRequestCreationViewModel : BaseViewModel, IPullRequestCreationV
                         notifications.ShowError(error?.Message ?? ex.Message);
                         return Observable.Empty<IPullRequestModel>();
                     }))
-
```
Comment (shana) [inline src/GitHub.App/ViewModels/PullRequestCreationViewModel.cs]: Apologies for the nitpick, but could you remove this change? Pure whitespace changes can potentially cause conflicts, we try to keep those separate from other changes.

**Comment by paladique** [comment]
👋 @jswietek Thanks so much for this! I'm working on an issue similar to this in #665 so I'll retarget this to my branch, merge, and continue from there.

**Pull request merged**